backend/src/controllers/CMSTutorialController.ts

```diff
@@ -20,7 +20,7 @@ export default class CMSTutorialController {
      * @param {Response} res our res obj
      */
     public static getAllTutorials(req: Request, res: Response): void {
-        TutorialUpdateService.getAllDocuments<Tutorial>("tutorials").then(
+        DbUpdateService.getAllDocuments<Tutorial>("tutorials").then(
```

Comment:
Lot better.

Reply:
When I renamed it first time I didn't notice I made it for general db queries lol mybad!!